.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+<!--
+IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION WE MIGHT CLOSE YOUR ISSUE WITHOUT INVESTIGATING
+-->
+
+**I'm submitting a ...**  (check one with "x")
+```
+[ ] bug report => search github for a similar issue or PR before submitting
+[ ] feature request
+[ ] support request => You can submit support request here
+```
+
+**Current behavior**
+<!-- Describe how the bug manifests. -->
+
+**Expected behavior**
+<!-- Describe what the behavior would be without the bug. -->
+
+**Minimal reproduction of the problem with instructions**
+
+(you can use [Codepen skate template](http://codepen.io/Hotell/pen/apqXYb) as a starting point)
+<!--
+If the current behavior is a bug or you can illustrate your feature request better with an example,
+please provide the *STEPS TO REPRODUCE* and if possible a *MINIMAL DEMO* of the problem via
+Codepen or similar
+-->
+
+**What is the motivation / use case for changing the behavior?**
+<!-- Describe the motivation or the concrete use case -->
```

Comment:
This implies a change, but it could be an addition. I wonder if this could just say "Use cases, if any".